### ISSUE TITLE
Looking for more Series 3 compounds and kinase inhibitors

### ISSUE THREAD

**Issue by holeung** [OpenSourceMalaria/OSM_To_Do_List#573]
Our partners at the Univ. Kansas have successfully produced Pf CLK1 and PKA-R in milligram quantities for ligand binding and structural studies. We previously predicted these two proteins as the most likely targets for the most potent Series 3 compounds.  Even if they are not, we now have the opportunity for target-based drug design against two proteins known to be essential for Plasmodium.

Our lab has the PKIS (North Carolina/SGC) library of kinase inhibitors. We will perform docking studies with other kinase inhibitor libraries. Please contact us if you have access to any promising compounds.

**Comment by holeung** [comment]
The relevant PDB codes:

CLK1: 3LLT
PKA-R: 5KBF

**Comment by mattodd** [comment]
That's fantastic news. So, we can now evaluate whether these are targets of S3..?

**Comment by holeung** [comment]
Yes, please send OSM-S-106 and OSM-S-137. Let's start with the potent compounds first.

**Comment by mattodd** [comment]
Well, it seems to me to be a good idea to test OSM-S-106 as the experiment (we know it's active vs parasite), and then perhaps an inactive compound? Maybe also a weakly active compound?

**Comment by holeung** [comment]
Weakly active sounds good.

**Comment by mattodd** [comment]
David Drewry from UNC (OpenSourceMalaria/Series3#8) put me on to [this preprint](https://www.biorxiv.org/content/early/2018/08/31/404459) from Andrew Tobin and others about the validation of PfCLK3. Might be useful and relevant.

**Comment by holeung** [comment]
We're ready to test. Please send compounds to me when ready. I believe you have my shipping address. Thank you.

**Comment by mbhebhe** [comment]
I will go have a look and see what other weakly active or inactive compounds we still have. As for OSM-S-137, we don't have any stock but I should be able to synthesize it. I need to couple the two halves together. 
@holeung how much do you need, of each compound?

**Comment by holeung** [comment]
Ideally, 5 mg.

**Comment by MFernflower** [comment]
@holeung do not know how useful this may be but this clk inhbitor looks an awful lot like our s3 hit https://www.sigmaaldrich.com/catalog/product/sigma/t5575?lang=en&region=US

**Comment by MFernflower** [comment]
Not as far as I know

**Comment by MFernflower** [comment]
(Apologies for the repost)

@holeung do not know how useful this may be but this clk inhbitor looks somewhat like our s3 hit!!!! https://www.sigmaaldrich.com/catalog/product/sigma/t5575?lang=en&region=US

**Comment by holeung** [comment]
@MFernflower: Great, we can test it.

**Comment by MFernflower** [comment]
@holding have you planned on studying Imatinib? It's been shown to have antimalarial properties

**Comment by holeung** [comment]
Imatinib is a tyrosine kinase inhibitor. CLK1 is a "dual-specificity" kinase, and can phosphorylate serine, threonine, or tyrosine. I don't think there is strong evidence that imatinib blocks dual-specificity kinases. However, I found a clinically used kinase inhibitor, sunitinib, which is active against human CLK1 and has also shown antimalarial activity. Through docking, I found it compatible with Pf CLK1. I will test sunitinib experimentally.

https://www.nature.com/articles/s41467-017-01345-2

**Comment by MFernflower** [comment]
My bad

**Comment by MFernflower** [comment]
@mbhebhe have other variations of the piperazinyl-thienopyrmidine been explored? eg:
![4373443](https://user-images.githubusercontent.com/3164942/46985671-8beea480-d0b9-11e8-9c4b-e015b7ed5a5e.png)

**Comment by mattodd** [comment]
@MFernflower https://github.com/OpenSourceMalaria/Series3/wiki/All-Analogs-Evaluated-to-Date

**Comment by MFernflower** [comment]
@holeung I also happend across this compound: https://www.sigmaaldrich.com/catalog/product/sigma/sml0924?lang=en&region=US

Might want to dock it to malaria CLK before ordering however?

**Comment by holeung** [comment]
Thanks, @MFernflower. Docking seems okay. I'll look into ordering and testing it.

**Comment by mattodd** [comment]
Really? That compound docks well? Amazing.

Exciting update today from the US team breeding parasites resistant to OSM-S-106:

"We have good news regarding your OSM-S-106 compound. We are cloning resistant parasites at the moment. We are hoping to mail gDNA within the next month to Elizabeth’s lab for WGS.
I am excited to learn you are already expressing putative protein targets. I will keep you updated on our progress."

So we may be getting confluence here on predictive, biochemical and genetic approaches.

**Comment by MFernflower** [comment]
Found a rather interesting paper today: @holeung https://www.sciencedirect.com/science/article/pii/S0960894X13002837?via%3Dihub

**Comment by MFernflower** [comment]
@mattodd @holeung This natural product looks an awful lot like our s3 hit! http://www.chemspider.com/Chemical-Structure.27024217.html

**Comment by holeung** [comment]
@MFernflower: That's intriguing. I'm not sure why I hadn't seen that earlier. Caboxamycin is described as a phosphodiesterase (opens cAMP or cGMP) inhibitor. I'll look into the plasmodium phosphodiesterase literature (will take me some time). We are currently doing biochemistry work with PKA-R, for which we had predicted OSM-S-106 binds to the cAMP binding site.

**Comment by MFernflower** [comment]
It probally acts as an mimic of camp that cannot be cleared from the camp
binding site of the target enzyme

In theroy it's 1 to 1 inhibition ratio - 1 molecule of carboxymycin
disables single molecule of target enzyme

Does it fit into the camp site of pkar?

**Comment by holeung** [comment]
We received the compounds from Sydney. I can't find a commercial source of caboxamycin. Anyone interested in synthesis?

**Comment by drc007** [comment]
http://www.ichemical.com/products/906095-21-4.html

**Comment by MedChemProf** [comment]
@holeung What time frame do you need for the Caboxamycin?

**Comment by holeung** [comment]
> http://www.ichemical.com/products/906095-21-4.html

They quoted me $1900.

**Comment by holeung** [comment]
> @holeung What time frame do you need for the Caboxamycin?

Not in a hurry. Interested?

**Comment by MedChemProf** [comment]
@holeung It is only 2-3 steps depending on which of the starting materials I can get a hold of. Finding the manpower is a little more problematic at the moment. Give me a few days to see if I can find a student. I just did not want to hold you up if we take longer than promised.

**Comment by MedChemProf** [comment]
@holeung I placed order for starting materials today. The Thanksgiving holiday may delay our start.

**Comment by MFernflower** [comment]
@MedChemProf do post your synthetic method should we ever wish to make a molecule combining features of both caboxamycin and OSM-S-106 - I.E NC1=CC=CC2=C1N=C(O2)C=2C=C(C=CC2)S(=O)(=O)N

**Comment by MFernflower** [comment]
@MedChemProf do post your synthetic method should @mbhebhe  ever wish to make the benzoxazole analogue of OSM-S-106!

Apologize for the repost

**Comment by MFernflower** [comment]
@holeung @mbhebhe  I have came up with a few molecules that combine features of caboxamycin and the s3 hit 
A docking study would be greatly appreciated provided you have the time to do so
![image](https://user-images.githubusercontent.com/3164942/48876168-bc1d1780-edca-11e8-8961-5f21f3be52ea.png)

**Comment by MedChemProf** [comment]
@holeung We have had a slow start, but we have just finished synthesizing the middle intermediate in the scheme shown below. We are going to scale up and bring a little more of this intermediate forward before completing the cyclization and deprotection steps.

![caboxamycinsynth](https://user-images.githubusercontent.com/14179120/52859800-af733b80-30fb-11e9-974b-1fd97a20d316.jpg)

The notebook for the above first steps can be found here: https://mynotebook.labarchives.com/share/MCPHS%2520MedChem/OTI5LjV8MzY3Ny83MTUvVHJlZU5vZGUvOTg2MDEzOTc0fDIzNTkuNQ==

**Comment by MFernflower** [comment]
@medchemprof will samples of protected and unprotected caboxamycin be sent off for parasite eval? @mattodd

**Comment by MedChemProf** [comment]
Not planning on it since we were not sending it out for parasite evaluation.

**Comment by MedChemProf** [comment]
@holeung Just sending out another update on our progress (slow as it has been). We now have the benzyl protected Caboxamycin. (https://mynotebook.labarchives.com/share/MCPHS%2520MedChem/OTU0LjJ8MzY3Ny83MzQvVHJlZU5vZGUvOTAzMjU0MDgzfDI0MjIuMg==) We are now in-between semesters, but hopefully we will finish the final deprotection step in the next few weeks.

**Comment by mbhebhe** [comment]
@MedChemProf we were wondering if there are any updates on the synthesis of caboxamycin

**Comment by MedChemProf** [comment]
@mbhebhe My apologies for the delay. The student that was working on the synthesis actually did complete the synthesis, but during the final isolation after purification accidentally dropped the flask (compound and solvent) and we lost the compound. An attempt to extract the material from across the bench top did not recover the material. The student has just returned to campus this past week and has already completed the first step in the synthesis and is currently purifying. I will let you know how much of the material she has advanced in the synthesis (see above scheme, compound in box) following purification. We will make this a priority.

**Comment by MedChemProf** [comment]
@mbhebhe @holeung I will not know until Thursday and high field NMR confirmation, but we may have finished the synthesis of Caboxamycin. We ran into a few problems, but we made one last attempt. Two questions: Does anyone still want the material? If so, what is the minimum you need? (we did not have a lot of material prior to the last deprotection step. Please advise and thank you.

**Comment by holeung** [comment]
Great! Hopefully, @mbhebhe or one of her colleagues can test it first for biological activity. If there is any left, we could use 2-5 mg for testing. We're short-handed at the moment so will need some time to perform the testing.

**Comment by mbhebhe** [comment]
Hi all,
Hope you guys are holding up ok with the COVID-19 situation.

@MedChemProf that's great news. We were wondering if you were still in contact with the Broad Institute and if they could do the biological activity. If not, you could send to us and we would send to Dundee.
@holeung so the next step will be for you to test caboxamycin against PfCLK1? Is the PfCLK1 that you expressed still good? Hopefully the compounds that I sent you are still good. 

Don't know if you guys remember, we sent OSM-S-106 to UNC for a kinome [scan](https://github.com/OpenSourceMalaria/Series3/issues/17). We had a look at the results again and OSM-S-106 does not inhibit HsCLK1. With this in mind, are we expecting caboxamycin to be an inhibitor of PfCLK1? Is it an inhibitor of HsCLK1?

**Comment by holeung** [comment]
We will have to remake the PfCLK1, which shouldn't be difficult. But the university is shutting down all research labs. It feels like the whole country is powering down.

**Comment by holeung** [comment]
We can look into PfCDPK1 too.

**Comment by MedChemProf** [comment]
Hello @mbhebhe and @holeung, Please brace yourselves for the excuses. I am feeling largely defeated at this point. We purified the sample which was a single spot on tlc, but the student must have picked up an impurity from somewhere else (I am thinking possibly a dirty NMR tube. The NMR shows what I interpret as product with the additional impurity. We now only have 5mg remaining. If I had a reverse HPLC like I did in industry, it would be no problem to re-purify and get a LCMS confirmation and have more than enough to submit. I also have lost contact with my Broad contact for testing. The cherry on top of this SNAFU is that even if we tried to re-purify ourselves, the students no longer are on campus for the foreseeable future due to our University transition to online teaching.

**Comment by mbhebhe** [comment]
Yeah its the same over here. I guess we will resume after all this blows over. 
Hopefully you will be able to purify the sample. We can send it to Dundee for biological activity
